### ISSUE TITLE
Enabling Faithless resource pack results in "resource reload failed" error

### ISSUE THREAD

**Issue by Poopooracoocoo** [MehVahdJukaar/polytone#24]
### Environment
Prism Launcher 8.2
Minecraft 1.20.1
Fabric Loader 0.15.7
Fabric API 0.92.0
Polytone 1.14.13
[Faithless resource pack](https://www.curseforge.com/minecraft/texture-packs/faithless/files/5044649)

### Steps to reproduce
1. Launch instance
2. Click options
3. Click resource packs
4. Click arrow on Faithless resource pack
5. Apply changes

### Log
[latest.log](https://github.com/MehVahdJukaar/polytone/files/14645168/latest.log)

<details><summary>Details</summary>

```
[15:05:10] [main/INFO]: Loading Minecraft 1.20.1 with Fabric Loader 0.15.7
[15:05:10] [main/INFO]: Loading 59 mods:
	- fabric-api 0.92.0+1.20.1
	   |-- fabric-api-base 0.4.31+1802ada577
	   |-- fabric-api-lookup-api-v1 1.6.36+1802ada577
	   |-- fabric-biome-api-v1 13.0.13+1802ada577
	   |-- fabric-block-api-v1 1.0.11+1802ada577
	   |-- fabric-block-view-api-v2 1.0.1+1802ada577
	   |-- fabric-blockrenderlayer-v1 1.1.41+1802ada577
	   |-- fabric-client-tags-api-v1 1.1.2+1802ada577
	   |-- fabric-command-api-v1 1.2.34+f71b366f77
	   |-- fabric-command-api-v2 2.2.13+1802ada577
	   |-- fabric-commands-v0 0.2.51+df3654b377
	   |-- fabric-containers-v0 0.1.64+df3654b377
	   |-- fabric-content-registries-v0 4.0.11+1802ada577
	   |-- fabric-convention-tags-v1 1.5.5+1802ada577
	   |-- fabric-crash-report-info-v1 0.2.19+1802ada577
	   |-- fabric-data-attachment-api-v1 1.0.0+de0fd6d177
	   |-- fabric-data-generation-api-v1 12.3.4+1802ada577
	   |-- fabric-dimensions-v1 2.1.54+1802ada577
	   |-- fabric-entity-events-v1 1.6.0+1c78457f77
	   |-- fabric-events-interaction-v0 0.6.2+1802ada577
	   |-- fabric-events-lifecycle-v0 0.2.63+df3654b377
	   |-- fabric-game-rule-api-v1 1.0.40+1802ada577
	   |-- fabric-item-api-v1 2.1.28+1802ada577
	   |-- fabric-item-group-api-v1 4.0.12+1802ada577
	   |-- fabric-key-binding-api-v1 1.0.37+1802ada577
	   |-- fabric-keybindings-v0 0.2.35+df3654b377
	   |-- fabric-lifecycle-events-v1 2.2.22+1802ada577
	   |-- fabric-loot-api-v2 1.2.1+1802ada577
	   |-- fabric-loot-tables-v1 1.1.45+9e7660c677
	   |-- fabric-message-api-v1 5.1.9+1802ada577
	   |-- fabric-mining-level-api-v1 2.1.50+1802ada577
	   |-- fabric-model-loading-api-v1 1.0.3+1802ada577
	   |-- fabric-models-v0 0.4.2+9386d8a777
	   |-- fabric-networking-api-v1 1.3.11+1802ada577
	   |-- fabric-networking-v0 0.3.51+df3654b377
	   |-- fabric-object-builder-api-v1 11.1.3+1802ada577
	   |-- fabric-particles-v1 1.1.2+1802ada577
	   |-- fabric-recipe-api-v1 1.0.21+1802ada577
	   |-- fabric-registry-sync-v0 2.3.3+1802ada577
	   |-- fabric-renderer-api-v1 3.2.1+1802ada577
	   |-- fabric-renderer-indigo 1.5.1+1802ada577
	   |-- fabric-renderer-registries-v1 3.2.46+df3654b377
	   |-- fabric-rendering-data-attachment-v1 0.3.37+92a0d36777
	   |-- fabric-rendering-fluids-v1 3.0.28+1802ada577
	   |-- fabric-rendering-v0 1.1.49+df3654b377
	   |-- fabric-rendering-v1 3.0.8+1802ada577
	   |-- fabric-resource-conditions-api-v1 2.3.8+1802ada577
	   |-- fabric-resource-loader-v0 0.11.10+1802ada577
	   |-- fabric-screen-api-v1 2.0.8+1802ada577
	   |-- fabric-screen-handler-api-v1 1.3.30+1802ada577
	   |-- fabric-sound-api-v1 1.0.13+1802ada577
	   |-- fabric-transfer-api-v1 3.3.4+1802ada577
	   \-- fabric-transitive-access-wideners-v1 4.3.1+1802ada577
	- fabricloader 0.15.7
	   \-- mixinextras 0.3.5
	- java 17
	- minecraft 1.20.1
	- polytone 1.20-1.14.13
	   \-- net_objecthunter_exp4j 0.4.8
[15:05:11] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/anon/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[15:05:11] [main/INFO]: Compatibility level set to JAVA_16
[15:05:11] [main/INFO]: Compatibility level set to JAVA_17
[15:05:14] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.5).
[15:05:23] [Datafixer Bootstrap/INFO]: 188 Datafixer optimizations took 309 milliseconds
[15:05:27] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[15:05:28] [Render thread/INFO]: Setting user: Anon
[15:05:28] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[15:05:29] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
[15:05:31] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric (fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-tables-v1, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-networking-v0, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, polytone)
[15:05:32] [Worker-Main-1/INFO]: Found unifont_all_no_pua-15.0.06.hex, loading
[15:05:35] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[15:05:35] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[15:05:35] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Speakers (Realtek(R) Audio)
[15:05:35] [Render thread/INFO]: Sound engine started
[15:05:35] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[15:05:35] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[15:05:35] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[15:05:35] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[15:05:35] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[15:05:35] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[15:05:35] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[15:05:35] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[15:05:35] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[15:05:36] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[15:05:36] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[15:05:36] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[15:05:36] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[15:05:36] [Render thread/INFO]: Loaded GUI modifiers for: [] [] [] []
[15:05:50] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric (fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-tables-v1, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-networking-v0, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, polytone), file/Faithless.zip
[15:05:50] [Worker-Main-3/INFO]: Found unifont_all_no_pua-15.0.06.hex, loading
[15:05:50] [Worker-Main-1/WARN]: File minecraft:sounds/dig/stone5.ogg does not exist, cannot add it to event minecraft:block.stone.hit
[15:05:50] [Worker-Main-1/WARN]: File minecraft:sounds/dig/stone6.ogg does not exist, cannot add it to event minecraft:block.stone.hit
[15:05:50] [Worker-Main-1/WARN]: File minecraft:sounds/dig/stone5.ogg does not exist, cannot add it to event minecraft:block.stone.step
[15:05:50] [Worker-Main-1/WARN]: File minecraft:sounds/dig/stone6.ogg does not exist, cannot add it to event minecraft:block.stone.step
[15:05:56] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[15:05:56] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[15:05:56] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Speakers (Realtek(R) Audio)
[15:05:56] [Render thread/INFO]: Sound engine started
[15:05:57] [Render thread/INFO]: Created: 2048x1024x4 minecraft:textures/atlas/blocks.png-atlas
[15:05:57] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/signs.png-atlas
[15:05:57] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[15:05:57] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[15:05:57] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[15:05:57] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[15:05:57] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[15:05:57] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[15:05:57] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[15:05:57] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[15:05:57] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[15:05:57] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[15:05:57] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[15:05:57] [Render thread/INFO]: Caught error loading resourcepacks, removing all selected resourcepacks
java.util.concurrent.CompletionException: java.lang.IllegalStateException: Did not find any texture png for implicit colormap from block modifier minecraft:custom/lava
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:722) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at net.minecraft.class_4014.method_18365(class_4014.java:69) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1175) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:802) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
Caused by: java.lang.IllegalStateException: Did not find any texture png for implicit colormap from block modifier minecraft:custom/lava
	at net.mehvahdjukaar.polytone.block.BlockPropertiesManager.addModifier(BlockPropertiesManager.java:150) ~[polytone-1.20-1.14.13-fabric.jar:?]
	at net.mehvahdjukaar.polytone.block.BlockPropertiesManager.process(BlockPropertiesManager.java:124) ~[polytone-1.20-1.14.13-fabric.jar:?]
	at net.mehvahdjukaar.polytone.block.BlockPropertiesManager.process(BlockPropertiesManager.java:29) ~[polytone-1.20-1.14.13-fabric.jar:?]
	at net.mehvahdjukaar.polytone.utils.CompoundReloader.processTyped(CompoundReloader.java:54) ~[polytone-1.20-1.14.13-fabric.jar:?]
	at net.mehvahdjukaar.polytone.utils.CompoundReloader.apply(CompoundReloader.java:41) ~[polytone-1.20-1.14.13-fabric.jar:?]
	at net.mehvahdjukaar.polytone.utils.CompoundReloader.method_18788(CompoundReloader.java:11) ~[polytone-1.20-1.14.13-fabric.jar:?]
	at net.minecraft.class_4080.method_18790(class_4080.java:13) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718) ~[?:?]
	... 15 more
[15:05:57] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric (fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-tables-v1, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-networking-v0, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, polytone)
[15:05:57] [Worker-Main-2/INFO]: Found unifont_all_no_pua-15.0.06.hex, loading
[15:05:59] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[15:05:59] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[15:05:59] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Speakers (Realtek(R) Audio)
[15:05:59] [Render thread/INFO]: Sound engine started
[15:05:59] [Render thread/INFO]: Created: 2048x1024x4 minecraft:textures/atlas/blocks.png-atlas
[15:05:59] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/signs.png-atlas
[15:05:59] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[15:05:59] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[15:05:59] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[15:05:59] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[15:05:59] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[15:05:59] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[15:05:59] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[15:05:59] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[15:06:00] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[15:06:00] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[15:06:00] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[15:06:00] [Render thread/INFO]: Loaded GUI modifiers for: [] [] [] []
[15:06:50] [Render thread/INFO]: Stopping!

``` 

</details>

**Comment by MehVahdJukaar** [comment]
use new polytone version

**Comment by Poopooracoocoo** [comment]
I confirm that this issue has been resolved with the changes in https://github.com/MehVahdJukaar/polytone/commit/7d0af78bc7f54f8fab43b9bca8894c3eb162b6b2 that landed in Polytone 1.15.0. Thank you!
